amf-api-contract/shared/src/main/scala/amf/apicontract/internal/spec/common/Parameters.scala

```diff
@@ -33,6 +33,14 @@ case class Parameters(query: Seq[Parameter] = Nil,
     )
   }
 
+  def duplicates(inner: Parameters): List[Parameter] = {
```

Comment:
the name `duplicates` doesn't explain what the method does, maybe something like `findDuplicatesIn`